### PR TITLE
Annotate: Replace """ --> #

### PR DESCRIPTION
**Pull request by abh3po**

### CODE REVIEW

File: dev-requirements.txt [hunk 1]
```diff
@@ -0,0 +1,2 @@
+flake8
+pytest==4.3.0
```
Comment (yoshitha) [inline dev-requirements.txt:2]: What are dev-requirements? Specific to development? Should the dev install both requirements and dev-requirements?

File: tests/test_annotate.py [hunk 2]
```diff
@@ -1,27 +1,28 @@
...
-  assert result == '''#====== Schema information
+    database = "test.db"
+    create_database(database)
+    file_path = 'tests/fixture_models/tasks.py'
+    write_to_file(os.path.dirname(file_path),
+                  'tests/fixture_models/fixture_model_1.py')
+    result = open(file_path).read()
+    drop_database(database)
+    truncate_file(file_path)
+    print(result)
```
Comment (yoshitha) [inline tests/test_annotate.py]: x

File: tests/test_models.py [hunk 3]
```diff
@@ -1,77 +1,87 @@
-import os
-import sys
-sys.path.insert(0, os.path.abspath(os.path.join(os.path.dirname(__file__), '../')))
-from orator import DatabaseManager
-import sqlite3
-
-from annotatelib.models import(
+from annotatelib.models import (
```
Comment (yoshitha) [inline tests/test_models.py:1]: why did you move?

File: tests/test_models.py [hunk 4]
```diff
@@ -1,77 +1,87 @@
...
     }
-  db = DatabaseManager(config)
-  result = _get_indices_description_from_oject(db.get_schema_manager(), 'tasks')
-  print(result)
-  assert result == {'primary': {'is_unique?': True, 'is_primary?': True, 'columns': ['id']}}
-  drop_database(database)
+    db = DatabaseManager(config)
+    result = _get_indices_description_from_oject(
+        db.get_schema_manager(), 'tasks')
+    print(result)
```
Comment (yoshitha) [inline tests/test_models.py]: x

File: tests/test_annotate.py [hunk 5]
```diff
@@ -20,17 +20,16 @@ def test_write_to_file():
...
-id         integer                            primary_key
-name       text           not null
-priority   integer
-status_id  integer        not null
-project_id integer        not null
-begin_date text           not null
-end_date   text           not null
-
-"""
+  print(result)
```
Comment (yoshitha) [inline tests/test_annotate.py]: x

File: dev-requirements.txt (hunk 1 above)
Comment (abh3po) [reply dev-requirements.txt:2]: Yes.

File: tests/test_models.py (hunk 3 above)
Comment (abh3po) [reply tests/test_models.py:1]: Unused import I think it was the linter

Comment (abh3po) [reply tests/test_models.py:1]: it was the bracket, linter wanted it

**Review by yoshitha — APPROVED** [review]
